CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 - PHP: Constant propagation now has built-in knowledge of `escapeshellarg` and
   `htmlspecialchars_decode`, if these functions are given constant arguments,
   then Semgrep assumes that their output is also constant
+- When a `metavariable-comparison` has `strip: true` set,
```

Comment:
Better rebase and fix the changelog, otherwise when we merge this PR this entry will go under `0.83.0` which is already released.